inginious/frontend/templates/layout.html

```diff
@@ -140,6 +140,11 @@
                                 <a class="list-group-item list-group-item-action" href="{{ get_homepath() }}/queue"><i class="fa fa-dashboard fa-fw"></i> {{ _("Service status") }}</a>
                                 <a class="list-group-item list-group-item-action" href="{{ get_homepath() }}/logout" id="logoff_button"><i class="fa fa-sign-out"></i> {{ _("Log out") }}</a>
                             </div>
+                            <div class="list-group" style="bottom: 0; position: fixed; width: 250px; z-index: 1O">
```

Comment:
Ok it's now "working" but have some awful behavior with responsiveness. For some width, this do not show up. For some it displays badly and for some it's out of the menu. Have a look at responsiveness with developer tools.